coordinator/internal/types/get_task.go

```diff
@@ -2,15 +2,17 @@ package types
 
 // GetTaskParameter for ProverTasks request parameter
 type GetTaskParameter struct {
-	ProverHeight uint64 `form:"prover_height" json:"prover_height"`
-	TaskType     int    `form:"task_type" json:"task_type"`
-	VK           string `form:"vk" json:"vk"`
+	ProverHeight uint64   `form:"prover_height" json:"prover_height"`
+	TaskType     int      `form:"task_type" json:"task_type"`
+	VK           string   `form:"vk" json:"vk"`
+	VKs          []string `form:"vks" json:"vks"`
```

Comment:
we can add some comments here to differentiate prover versions for future maintenance (e.g., deprecate some fields). e.g., used for xxx prover.

Reply:
done